### ISSUE TITLE
Wrong ASCII warning when inputing Chinese

### ISSUE THREAD

**Issue by Moelf** [nscaife/linter-chktex#23]
Certain Chinese character like "水“ (water) is spotted incorrectly.
![image](https://cloud.githubusercontent.com/assets/5306213/11774881/cc054090-a274-11e5-9a92-1f4467b69414.png)

![image](https://cloud.githubusercontent.com/assets/5306213/11774867/c00e11fe-a274-11e5-8e94-e5e3c3038b7b.png)

**Comment by Arcanemagus** [comment]
Looks like this is an issue with `chktex` not this plugin.

While linting the following text:

``` latex
\pause\item It is widely used in academia (\textipa{/""\ae{}k@"di:mI9/}).
\pause\item It is widely used in academia (学术界 \textipa{/""\ae{}k@"di:mI9/}).
```

This linter shows this:
![image](https://cloud.githubusercontent.com/assets/427137/11882715/00554a84-a4c3-11e5-8fcc-cbd5bf3c4eac.png)

However this matches exactly what `chktex` is giving us:
![image](https://cloud.githubusercontent.com/assets/427137/11882753/378a34a6-a4c3-11e5-803c-b7546de13e1b.png)

**Comment by Moelf** [comment]
So........there's no way for us to do anything about this?

**Comment by Arcanemagus** [comment]
Not from here, you should file a bug over on the `chktex` issue tracker. It looks like the location of that for the most current version is here: https://savannah.nongnu.org/bugs/?group=chktex
